docs/openapi.yaml

```diff
@@ -672,6 +672,52 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Problem"
+  /institution-approval-report/{year}:
```

Comment:
@ketilaa and I originally discussed using path `/institution-report/{year}` for `FetchInstitutionReportHandler`, but this path is already used for retrieving organization aggregations (`FetchInstitutionStatusAggregationHandler`).
Therefore named path `/institution-approval-report/{year}`